sbt-tasks/test/com/commercetools/sunrise/theme/WebjarsFilesCopierTest.java

```diff
@@ -15,12 +15,19 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class WebjarsFilesCopierTest {
-    private static final String DEST_PATH = WebjarsFilesCopierTest.class.getClass().getResource("/com/commercetools/sunrise/theme").getPath();
+
+    private static final String DEST_PATH = WebjarsFilesCopierTest.class.getClass().getResource("/").getPath() + "webjarsFilesCopierDestFolder";
 
     @Before
     @After
-    public void setUp() throws Exception {
-        FileUtils.cleanDirectory(new File(DEST_PATH));
+    public void deleteFolderWithAllCopiedFiles() throws Exception {
+        final File directory = new File(DEST_PATH);
+        if (directory.exists()) {
+            FileUtils.cleanDirectory(directory);
+            if (!directory.delete()) {
```

Comment:
... I want to delete it too, so to leave the folder as clean as it was before the test ran.